text/0000-javascript-module-api.md

```diff
@@ -0,0 +1,1093 @@
+| `Ember.Helper.helper`                 | `import { helper } from "@ember/component/helper"`                         |
+| `Ember.HistoryLocation`               | `import HistoryLocation from "@ember/routing/history-location"`            |
+| `Ember.Location`                      | `import Location from "@ember/routing/location"`                           |
+| `Ember.Map`                           | `import EmberMap from "@ember/map"`                                        |
+| `Ember.MapWithDefault`                | `import MapWithDefault from "@ember/map/with-default"`                     |
+| `Ember.Mixin`                         | `import Mixin from "@ember/object/mixin"`                                  |
+| `Ember.MutableArray`                  | `import MutableArray from "@ember/array/mutable"`                          |
+| `Ember.NoneLocation`                  | `import NoneLocation from "@ember/routing/none-location"`                  |
+| `Ember.Object`                        | `import EmberObject from "@ember/object"`                                  |
+| `Ember.RSVP`                          | `import RSVP from "rsvp"`                                                  |
+| `Ember.Resolver`                      | `import Resolver from "@ember/application/resolver"`                       |
+| `Ember.Route`                         | `import Route from "@ember/routing/route"`                                 |
+| `Ember.Router`                        | `import Router from "@ember/routing/router"`                               |
+| `Ember.Service`                       | `import Service from "@ember/service"`                                     |
+| `Ember.String.camelize`               | `import { camelize } from "@ember/string"`                                 |
+| `Ember.String.capitalize`             | `import { capitalize } from "@ember/string"`                               |
+| `Ember.String.classify`               | `import { classify } from "@ember/string"`                                 |
+| `Ember.String.dasherize`              | `import { dasherize } from "@ember/string"`                                |
+| `Ember.String.decamelize`             | `import { decamelize } from "@ember/string"`                               |
+| `Ember.String.fmt`                    | `import { fmt } from "@ember/string"`                                      |
+| `Ember.String.htmlSafe`               | `import { htmlSafe } from "@ember/string"`                                 |
+| `Ember.String.loc`                    | `import { loc } from "@ember/string"`                                      |
+| `Ember.String.underscore`             | `import { underscore } from "@ember/string"`                               |
+| `Ember.String.w`                      | `import { w } from "@ember/string"`                                        |
+| `Ember.TextArea`                      | `import TextArea from "@ember/component/text-area"`                        |
```

Comment:
I prefer it singular, but other things such as `utils` so that is likely something we may want to address..

cc @tomdale